examples/ERP/classify_P300_bi.py

```diff
@@ -104,19 +112,38 @@
     LDA(solver="lsqr", shrinkage="auto"),  # you can use other classifiers
 )
 
-print("Total pipelines to evaluate: ", len(pipelines))
+# We cache the results on Firebase.
+# But you can skip all cache functions bellow if you want.
+caches = generate_caches(datasets, pipelines)
+
+# This method remove a subject in a dataset if we already have evaluated
+# all pipelines for this subject.
+# Therefore we will use a copy of the original datasets.
+filter_subjects_with_all_results(caches, copy_datasets, pipelines)
```

Comment:
filter_subjects_with_all_results - name can be improved. 
it can be "filter_subjects_with_existing_results" ?